components/lib/crash-sentry/src/main/java/mozilla/components/lib/crash/sentry/SentryService.kt

```diff
@@ -38,7 +40,8 @@ class SentryService(
     private val tags: Map<String, String> = emptyMap(),
     private val environment: String? = null,
     private val sendEventForNativeCrashes: Boolean = false,
-    private val sentryProjectUrl: String? = null,
+    private val sendCaughtExceptions: Boolean = false,
+    private val sentryProjectUrl: String? = null
```

Comment:
Nit: Could we `sendCaughtExceptions` to the end of the parameters just to avoid breaking consumers that are not using named parameters? 😅

Reply:
Removed the default, forcing consumers to make a selection.